docs/fastddsgen/usage/usage.rst

```diff
@@ -53,6 +53,12 @@ Where the option choices are:
 
    * - Option
      - Description
+   * - -cdr
+     - Sets the Fast-CDR version used to generate types source code. |br|
+       Values: |br|
+       - v1. Generate source code for Fast-CDR v1. |br|
```

Comment:
The official name is Fast CDR. As far as I know, the only official name with an hyphen is Fast DDS-Gen. The hyphen never appears between Fast and the corresponding product.

Reply:
Done in 7c4c331